### ISSUE TITLE
os.rename raises an error when copying between different FS

### ISSUE THREAD

**Issue by draftcode** [macisamuele/language-formatters-pre-commit-hooks#82]
Looks like it's easy to fix with https://docs.python.org/3/library/shutil.html#shutil.move per https://stackoverflow.com/questions/42392600/oserror-errno-18-invalid-cross-device-link

```
Downloading https://github.com/google/google-java-format/releases/download/v1.10.0/google-java-format-1.10.0-all-deps.jar
Traceback (most recent call last):
  File "/home/draftcode/.cache/pre-commit/repop2cp8yu_/py_env-python3.9/lib/python3.9/site-packages/language_formatters_pre_commit_hooks/pretty_format_java.py", line 37, in __download_google_java_formatter_jar
    return download_url(url_to_download, "google-java-formatter{version}.jar".format(version=version))
  File "/home/draftcode/.cache/pre-commit/repop2cp8yu_/py_env-python3.9/lib/python3.9/site-packages/language_formatters_pre_commit_hooks/utils.py", line 75, in download_url
    os.rename(tmp_file_name, final_file)
OSError: [Errno 18] Invalid cross-device link: '/tmp/tmp_yncpea2' -> '/home/draftcode/.cache/pre-commit/google-java-formatter1.10.0.jar'
```

**Comment by macisamuele** [comment]
This issue was already fixed by #69 but not yet released.

Could you please cross-check if the problem is gone while using the current version in `master` (`pip install https://github.com/macisamuele/language-formatters-pre-commit-hooks/archive/f581e56ae511b3c35b6c6856b4994f9e9f08c8ad.zip`)?

**Comment by draftcode** [comment]
Oh. OK. It worked. I thought I tested against HEAD, but apparently I didn't.
